<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Strong Password Generator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;Titillium Web&quot;;</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="layout_header">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="lb_main_icon">
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Resources.qrc">:/assets/lock_icon.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_main">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Titillium Web&quot;;</string>
        </property>
        <property name="text">
         <string>Strong Password Generator</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="layout_main">
      <item>
       <layout class="QHBoxLayout" name="length_layout">
        <item>
         <widget class="QLabel" name="lb_pslength">
          <property name="styleSheet">
           <string>font: 11pt &quot;Titillium Web&quot;;</string>
          </property>
          <property name="text">
           <string>Password Length:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider">
          <property name="minimum">
           <number>4</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="sliderPosition">
           <number>20</number>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="symbols_layout">
        <item>
         <widget class="QLabel" name="lb_pssymbols">
          <property name="styleSheet">
           <string>font: 11pt &quot;Titillium Web&quot;;</string>
          </property>
          <property name="text">
           <string>Use Symbols:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_pssymbols">
          <property name="maximumSize">
           <size>
            <width>92</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string>font: 11pt &quot;Titillium Web&quot;;</string>
          </property>
          <property name="text">
           <string>(@#$ etc)</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">optionsButtons</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="numbers_layout">
        <item>
         <widget class="QLabel" name="lb_psnumbers">
          <property name="styleSheet">
           <string>font: 11pt &quot;Titillium Web&quot;;</string>
          </property>
          <property name="text">
           <string>Use Numbers:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_psnumbers">
          <property name="maximumSize">
           <size>
            <width>92</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string>font: 11pt &quot;Titillium Web&quot;;</string>
          </property>
          <property name="text">
           <string>123456</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">optionsButtons</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="uppercase_layout">
        <item>
         <widget class="QLabel" name="lb_psuppercase">
          <property name="styleSheet">
           <string>font: 11pt &quot;Titillium Web&quot;;</string>
          </property>
          <property name="text">
           <string>Use Uppercase characters:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_psuppercase">
          <property name="maximumSize">
           <size>
            <width>92</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string>font: 11pt &quot;Titillium Web&quot;;</string>
          </property>
          <property name="text">
           <string>ABCDEF</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">optionsButtons</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="lowercase_layout">
        <item>
         <widget class="QLabel" name="lb_pslowercase">
          <property name="styleSheet">
           <string>font: 11pt &quot;Titillium Web&quot;;</string>
          </property>
          <property name="text">
           <string>Use Lowercase characters:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_pslowercase">
          <property name="maximumSize">
           <size>
            <width>92</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string>font: 11pt &quot;Titillium Web&quot;;</string>
          </property>
          <property name="text">
           <string>abcdef</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">optionsButtons</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layout_generate">
      <item>
       <widget class="QPushButton" name="btn_copy">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Titillium Web&quot;;</string>
        </property>
        <property name="text">
         <string>Copy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_generate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string>font: 11pt &quot;Titillium Web&quot;;</string>
        </property>
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordEdit">
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_status">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Password Copied to Clipboard</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>407</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="optionsButtons">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
